configuration/multimedia

```diff
@@ -1,136 +1,160 @@
 ---
 layout: doc
-title: How to play multimedia content
-permalink: /doc/config/multimedia/
+title: Multimedia
+permalink: /doc/multimedia/
+redirect_from:
+- /en/doc/multimedia/
+- /doc/Multimedia/
+- /wiki/Multimedia/
 ---
 
-This short howto will explain how you can use Spotify, Netflix or Amazon Prime under Qubes OS.
-This steps have been tested by providing an own "multimedia" App VM which is based on the Qubes OS Debian 8 Template VM.
+Multimedia
+==========
 
-Listen to Spotify
-=================
-Install-Howto: https://www.spotify.com/de/download/linux/
+In order to choose Qubes OS as your primary OS it must be able to do all tasks, including playing multimedia content.
```

Comment:
This is obviously not true.

Reply:
Ok, there might be different opinions on this topic. I would like to do all tasks that I can do with a computer within Qubes, which includes also multimedia.
Maybe we strip this out:

OLD: In order to choose Qubes OS as your primary OS it must be able to do all tasks, including playing multimedia content.

NEW: You can consolidate most of your media streaming tasks into one "multimedia" App-VM, which can based on an own multimedia template.

Reply:
Better, but I think "an own" might be a grammatical mistake.